tests/requirements.txt

```diff
@@ -1,8 +1,9 @@
-jinja2
-pyyaml
+jinja2 >= 3.0.0
+pyyaml >= 5.1
 resolvelib < 1.1.0
 sphinx == 5.3.0
-sphinx-notfound-page
-sphinx-ansible-theme
-rstcheck < 6  # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
+sphinx-notfound-page == 0.8.3
+sphinx-ansible-theme == 0.10.2
+rstcheck == 5.0.0  # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
 antsibull-docs == 2.2.0  # currently approved version
+antsibull-docs-parser  == 1.0.0
```

Comment:
@felixfontein Do you think it is necessary to add `antsibull-docs-parser` here?

Reply:
I think it is.